ui/src/pages/Tenant/helper.ts

```diff
@@ -175,29 +175,38 @@ export function checkIsSame(
   return true;
 }
 
-function findMinValue(
-  key: 'availableCPU' | 'availableLogDisk' | 'availableMemory',
-  resources: API.ServerResource[],
-) {
-  if (!resources.length) return [];
-  return resources.sort((pre, cur) => cur[key] - pre[key])[0][key];
-}
-
 export function findMinParameter(
   zones: string[],
   essentialParameter: API.EssentialParametersType,
 ): MaxResourceType {
-  
-  const { obServerResources } = essentialParameter;
-  const selectResources = obServerResources.filter((resource) =>
-    zones.includes(resource.obZone),
-  );
-  
-  return {
-    maxCPU: findMinValue('availableCPU', selectResources),
-    maxLogDisk: findMinValue('availableLogDisk', selectResources),
-    maxMemory: findMinValue('availableMemory', selectResources),
-  };
+  let result: MaxResourceType = {};
+  for (let zone of zones) {
+    if (Object.keys(result).length === 0) {
+      result = {
+        maxCPU: essentialParameter.obZoneResourceMap[zone]['availableCPU'],
+        maxLogDisk:
+          essentialParameter.obZoneResourceMap[zone]['availableLogDisk'],
+        maxMemory:
+          essentialParameter.obZoneResourceMap[zone]['availableMemory'],
+      };
+    } else {
+      result = {
+        maxCPU: Math.max(
+          result.maxCPU!,
+          essentialParameter[zone]['availableCPU'],
+        ),
+        maxLogDisk: Math.max(
+          result.maxLogDisk!,
+          essentialParameter[zone]['availableCPU'],
```

Comment:
Wrong field here, you should access diskSize and memory.

Reply:
ok